test/apps/iodemo/io_demo.cc

```diff
@@ -657,8 +677,30 @@ class DemoServer : public P2pDemoCommon {
         }
     }
 
+private:
+    void update_state() {
+        _curr_state.n_conn_requested = n_conn_requested();
+        _curr_state.n_conn_accepted  = n_conn_accepted();
+    }
+
+    void save_state() {
+        _prev_state = _curr_state;
+    }
+
+    void report_state() {
+        update_state();
+        LOG << "active connections: " << n_connections()
+            << ", requested: "
+            << _curr_state.n_conn_requested - _prev_state.n_conn_requested
+            << ", accepted: "
+            << _curr_state.n_conn_accepted - _prev_state.n_conn_accepted;
```

Comment:
IMO need to print smth like this:
"read: 100 ops, write: 105 ops, active connections: 10"

Reply:
fixed